### PR TITLE
Add Mediapipe Tensors support

### PR DESCRIPTION
**Pull request by atobiszei**
* Additionally fix sporadic in model manager test by removing unnecessary assertion
* Minor tweaks in MP execution logs
* TfLite Tensors support in (de)serialization is turned off for 2023.1

Need to raise threshold due to cppclean complaining that
we need to have static specifier in template specialization which is not
possible (this specifier can be added only in template declaration).

CVS-116277

### CODE REVIEW

File: WORKSPACE
Comment (atobiszei) [inline WORKSPACE:92]: Commit will be bumped again after related MP fork repo MR will be merged to main

**Review by rasapala — APPROVED** [review]
LGTM

File: WORKSPACE
Comment (dkalinowski) [inline WORKSPACE:216]: There is commit by @michalkulakowski added today that fixes cpp benchmark app building. It also uses TF. Michal needed to bring all patches from OVMS to the benchmark app. Can we confirm this commit does not break cpp benchmark app again?

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:495]: What about 0-dim requests?

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:231]: Wrong message, zero is allowed in line 229

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:238]: Needs testing if passing nullptr as buffer is supported (0-data tensors)

File: src/test/mediapipeflow_test.cpp
Comment (dkalinowski) [inline src/test/mediapipeflow_test.cpp:176]: We should also check scalar and 0-data scenario, for example method for creating 0-data tensor differs from creating one with data in OpenVINO

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:157]: We should allow and test zeros in shape

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:778]: 3x `{}`, but 2 parameters - will crash at some point. Do we have test executing this code?

File: src/test/c_api_stress_tests.cpp
Comment (dkalinowski) [inline src/test/c_api_stress_tests.cpp:281]: This is probably already fixed on main - PR not rebased?

File: src/test/mediapipe/config_mp_passthrough.json
Comment (dkalinowski) [inline src/test/mediapipe/config_mp_passthrough.json:2]: This test will probably fail when I merge my commit.
My commit fixes the issue with existence of `subconfig.json` in tests which adds dummy model.
Here you want to use dummy model but do not load dummy model in line 2.
Keep it in mind before merging PR

File: src/test/mediapipeflow_test.cpp
Comment (atobiszei) [reply src/test/mediapipeflow_test.cpp:176]: TfLite Support is enabled only in calculator - not in OVMS

File: src/test/mediapipe/config_mp_passthrough.json
Comment (atobiszei) [reply src/test/mediapipe/config_mp_passthrough.json:2]: Here test uses subconfig.json

File: src/test/c_api_stress_tests.cpp
Comment (atobiszei) [reply src/test/c_api_stress_tests.cpp:281]: Fix was cherry-picked on this branch as well

File: WORKSPACE
Comment (atobiszei) [reply WORKSPACE:92]: As soon as I have approvals.

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (atobiszei) [inline src/mediapipe_internal/mediapipegraphexecutor.cpp:231]:
```suggestion
                ss << "Negative dimension size is not acceptable: " << tensorShapeToString(requestInputItr->shape()) << "; input name: " << requestedName;
```

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (atobiszei) [reply src/mediapipe_internal/mediapipegraphexecutor.cpp:238]: TfLite tensors in OVMS (de)serialization will not be supported in this release

Comment (dkalinowski) [reply src/mediapipe_internal/mediapipegraphexecutor.cpp:238]: So why do we need this code?

File: WORKSPACE
Comment (atobiszei) [reply WORKSPACE:216]: Benchmark builds OK with this MR without additional changes.

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [reply src/mediapipe_internal/mediapipegraphexecutor.cpp:157]: Is it scheduled in next task, next release or?

File: src/mediapipe_internal/mediapipegraphexecutor.cpp
Comment (dkalinowski) [reply src/mediapipe_internal/mediapipegraphexecutor.cpp:515]: Is it scheduled in next task, next release or?

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**
